packages/valaxy-theme-yun/components/YunConfig.vue

```diff
@@ -7,7 +7,7 @@ const { t } = useI18n()
 
 <template>
   <div>
-    <button class="yun-icon-btn" :title="t('button.toggle_dark')" :style="{ color: isDark ? '' : '#f1cb64' }" @click="toggleDark()">
+    <button class="yun-icon-btn" :title="t('button.toggle_theme')" :style="{ color: isDark ? '' : '#f1cb64' }" @click="toggleDark()">
```

Comment:
Regarding the naming, I refer to here, the same theme has both light and dark modes, so I think `toggle_dark` is reasonable.

https://github.com/antfu/vitesse/blob/52fdb6dd64ca7a2dabc6c4e4405b2ffd91ec47cc/locales/en.yml#L6

Reply:
But when it is dark and you want to toggle it into light, isn't `toggle_dark` a little weird?

Reply:
Or let's refer to vitepress, which uses `toggle dark mode` as secondary text.

So I guess toggle_dark should be semantic.

https://github.com/vuejs/vitepress/blob/00fe8092d9e097d2dd24c06787fcb740310bdda7/src/client/theme-default/components/VPSwitchAppearance.vue#L46

Reply:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/15975785/175504073-00528ee9-3752-4c08-8e3b-44408ed627b2.png">
<img width="265" alt="image" src="https://user-images.githubusercontent.com/15975785/175504104-da049469-9c2d-4f63-b015-db0376b5ac82.png">

Referenced from storybook.

Reply:
Furthermore, I think if this is going to be a general base application to the outside world, we should avoid creating misunderstanding by emphasizing the actual meaning here for example, when it is `dark` mode, the title should be `switch to light`, otherwise it should be `switch to dark`, since the screenreader won't know whether they are in `dark` or `light`

Reply:
My opinion is that the text should be explicit.